cloudferrylib/os/actions/convert_compute_to_image.py

```diff
@@ -42,6 +45,7 @@ def run(self, info=None, **kwargs):
         image_resource = self.cloud.resources[utl.IMAGE_RESOURCE]
         storage_resource = self.cloud.resources[utl.STORAGE_RESOURCE]
         compute_ignored_images = {}
+        missed_images = {}
```

Comment:
Very minor, but much better reading: the variable name is misspelt, should be `missing_images`.